docs/7.migration/1.overview.md

```diff
@@ -1,17 +1,17 @@
 # Overview
 
-Nuxt 3 is a complete rewrite of Nuxt 2, and also based on a new set of underlying technologies.
+Nuxt 3 is a complete rewrite of Nuxt 2, based on a new set of underlying technologies.
 
-That means there will be significant changes when migrating a Nuxt 2 app to Nuxt 3, although you can expect migration to become more straightforward as we move toward a stable release.
+That means there will be significant changes when migrating a Nuxt 2 app to Nuxt 3, although you can expect the migration to become more straightforward as we move toward a stable release.
```

Comment:
No longer relevant, unless you prefer to keep the first part of the sentence ("That means there will be significant changes when migrating a Nuxt 2 app to Nuxt 3.") and append it to the one above it.
```suggestion
```